TextileIPFSNativeModule.js

```diff
@@ -74,11 +74,15 @@ export default {
     return result
   },
 
-  getPhotoData: async function (path: string): string {
-    const result = await TextileIPFS.getPhotoData(path)
+  getHashData: async function (hash: string, path: string): string {
+    const result = await TextileIPFS.getHashData(hash + path)
     return result
   },
 
+  syncGetHashData: function (hash: string, path: string): string {
```

Comment:
We should just remove all sync functions since they don't work for Android. Can deal with this in a later PR.

Reply:
yeah, nobody uses this anyway so can just drop now